Design/Usage/AlgorithmUsageStatistics.md

```diff
@@ -0,0 +1,82 @@
+#Overview
+
+We have several algorithms that are similar and would love to rationalise the code and make the choice of which algorithm to use easier for scientists.
+However we always face the problem that we never know how much an algorithm is used.  It was agreed at the SSC2015 the we could track the usage of algorithms alongside simple startups of Mantid.
+
+This design document should be considered as an extension of the previous implemented design [MeasureUsageStatistics](MeasureUsageStatistics.md).
+
+##Proposed Changes
+
+We propose to Centralise the sending of all Usage reports into a central service, and extend the current functinality to 
+allow usage reports of features within Mantid, where a feature might be:
+
+1. Execution of an algorithm, parent or child
+1. Startup of a Interface
+1. Usage of a particular feature in an interface
+
+Initially we would only intend to automatically track Algorithm usage, but other usage reports can be added by developer as they deem helpful.
+The contents of a Feature usage would be:
+
+* type - Algorithm, Interface, Feature
+* name - Identifying name, for algorithms this would be Algorithm and version
+* time - The datetime of execution start
+* duration - (optional) the execution duration
+* details - (optional) Further details - for algorithms this will be a properties string, or possibly the python string of the Algorithm call.
+
+These would be sent in addition to these fields in common with the startup Usage details:
+
+* User ID hash
```

Comment:
I think you would want to also have the mantid version and operating system as well